app/viewmodels/helpers/events/MovementEventHelper.scala

```diff
@@ -459,4 +471,235 @@ class MovementEventHelper @Inject()(
+          temporaryAuthorisationReferenceRow,
+          consigneeTrader.vatNumber.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorConsignee.vatNumber", _)),
+          consigneeTrader.address.map(address => summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorConsignee.address", renderAddress(address))),
+          consigneeTrader.eoriNumber.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorConsignee.eoriNumber", _))
+        )
+      )
+    }
+
+    optContent.getOrElse(Html(""))
+  }
+
+  def rorDestinationCard(event: MovementHistoryEvent)(implicit movement: GetMovementResponse, messages: Messages): Html = {
+    val optContent: Option[Html] = for {
+      reportOfReceipt <- movement.reportOfReceipt
+      deliveryPlaceTrader <- reportOfReceipt.deliveryPlaceTrader
+      if movement.destinationType != DestinationType.Export
+    } yield {
+      val ernRow: Option[SummaryListRow] =
+        if (movement.destinationType == DestinationType.TaxWarehouse) {
+          deliveryPlaceTrader.traderExciseNumber.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorDeliveryPlaceTrader.ern", _))
+        } else {
+          None
+        }
+
+      buildOverviewPartial(
+        headingId = None,
+        headingTitle = Some(s"movementHistoryEvent.${event.eventType}.rorDeliveryPlaceTrader.h2"),
+        cardTitleMessageKey = None,
+        cardFooterHtml = None,
+        summaryListRows = Seq(
+          deliveryPlaceTrader.traderName.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorDeliveryPlaceTrader.name", _)),
+          ernRow,
+          deliveryPlaceTrader.vatNumber.map(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorDeliveryPlaceTrader.vatNumber", _)),
+          deliveryPlaceTrader.address.map(address => summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorDeliveryPlaceTrader.address", renderAddress(address))),
+        )
+      )
+    }
+
+    optContent.getOrElse(Html(""))
+  }
+
+  def rorExportCard(event: MovementHistoryEvent)(implicit movement: GetMovementResponse, messages: Messages): Html = {
+    val optContent: Option[Html] = for {
+      reportOfReceipt <- movement.reportOfReceipt
+      if movement.destinationType == DestinationType.Export
+    } yield {
+      buildOverviewPartial(
+        headingId = None,
+        headingTitle = Some(s"movementHistoryEvent.${event.eventType}.rorExport.h2"),
+        cardTitleMessageKey = None,
+        cardFooterHtml = None,
+        summaryListRows = Seq(
+          // TODO: check if destinationOffice is correct
+          Some(summaryListRowBuilder(s"movementHistoryEvent.${event.eventType}.rorExport.customsOfficeCode", reportOfReceipt.destinationOffice))
+        )
+      )
+    }
+
+    optContent.getOrElse(Html(""))
+  }
+
+  def rorItemsCard(event: MovementHistoryEvent, ie818ItemModelWithCnCodeInformation: Seq[IE818ItemModelWithCnCodeInformation])
```

Comment:
Couldn't the existing `Items` information card be updated for this? I'd have thought this content would have been consistent everywhere?